library/core/src/char/methods.rs

```diff
@@ -1738,6 +1740,7 @@ impl EscapeDebugExtArgs {
 }
 
 #[inline]
+#[must_use]
```

Comment:
these are private functions, so I don't think it matters much whether they are must_use, it seems very unlikely to cause problems if they aren't